### ISSUE TITLE
+global-js and other exporters likely broken

### ISSUE THREAD

**Issue by justinbmeyer** [stealjs/steal-tools#177]
in `lib/build/helpers` there's a check like:

```
if( depLoad.name === npmUtils.pkg.main(pkg) ) {
```

This no longer works.  If the main module's name will be something like:

```
synjs@0.0.3#syn
```

npmUtils.pkg.main(pkg) will be something like:

```
syn
```

**Comment by justinbmeyer** [comment]
Only global-js seems broken.  Tested and fixed with: https://github.com/bitovi/steal-tools/commit/ec7b13709e5dcc6c4c14b9095e6d31e53a0f44db

**Comment by matthewp** [comment]
This is fixed in 0.7.1
